src/core/lib/transport/transport.h

```diff
@@ -32,6 +32,12 @@
 #include "src/core/lib/transport/metadata_batch.h"
 
 /* forward declarations */
```

Comment:
Please move this comment down below the new lines you've added, since the new lines are not forward declarations.